third_party/maven_install.json

```diff
@@ -2991,6 +2991,50 @@
                 "sha256": "db06b388994733ed3a678819fc91b572a92abdd7277fbb856a48969f6df6ecf1",
                 "url": "https://maven.google.com/androidx/sqlite/sqlite/2.1.0/sqlite-2.1.0-sources.jar"
             },
+            {
+                "coord": "androidx.test.espresso:espresso-accessibility:3.1.0",
```

Comment:
Do you know why these are needed now? Is it because of a merge with develop?

Reply:
Yes, right, I took a pull from develop and this dependency was added.